subset/pentests/brute_client.py

```diff
@@ -0,0 +1,79 @@
+import socket, sys, json
+
+user_data=open('user_list.json')
+user_list = json.load(user_data)
+user_data.close()
+
+#username=['root','admin','manager']
+username= user_list['username'].split(",")
+
+#password=['root','pass','friend']
+password= user_list['password'].split(",")
+
+arguments = sys.argv
+
+print 'brute_client test begin'
+print 'address:' + arguments[1]
+print 'port:' + arguments[2]
+print 'reportFile:' + arguments[3]
+
+#address = '127.0.0.1'
+address = str(arguments[1])
+
+#port = 10000
+port = int(arguments[2])
+
+report_file = str(arguments[3])
+data_length =64
+
+expected=['login:','Password:','Last login:','Login incorrect','Connection closed by foreign host.']
+expected1=['username:','password:','success','login failed','Connection closed by foreign host.']
+
+sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+server_address = (address,port)
+
+sock.connect(server_address)
+
+login_report='\n'
+u=0
+p=0
+
+def _write_report():
+    file = open(report_file, "w")
+    file.write(login_report)
+
+try:
+    while True:
+        data = sock.recv(data_length)
+        if data:
+            print data
```

Comment:
I think this is the python2 version -- just FYI should try to use python3 in the future.

Reply:
noted, will update to python 3 for future versions. Sorry didn't realize we were using a beta release of python